media/src/project/project-map-sidebar.tsx

```diff
@@ -114,31 +213,85 @@ const AddEventPanel = ({
     );
 };
 
+interface DetailEventPanelProps {
+    activeLayer: number | null;
+    activeEventDetail: LayerEventDatum | null;
+    setActiveEventDetail(d: LayerEventDatum | null): any;
+    activeEventEdit: LayerEventDatum | null;
+    setActiveEventEdit(d: LayerEventDatum | null): any;
+    deleteEvent(pk: number, layerPk: number): any;
+}
+
+const DetailEventPanel = ({
+    activeLayer, activeEventDetail, setActiveEventDetail, activeEventEdit,
+    setActiveEventEdit, deleteEvent}: DetailEventPanelProps) => {
+    const [showMenu, setShowMenu] = useState<boolean>(false);
+
+    const handleDelete = (e: React.MouseEvent) => {
+        e.preventDefault();
```

Comment:
Delete events should come with "Are you sure" messaging (ideally). No need to add for this PR, but Z should likely add it to the wireframes.

Reply:
Ok, I added a TODO to remind ourselves, as well.